model/src/w3adatmd.F90

```diff
@@ -2233,10 +2221,14 @@ SUBROUTINE W3XDMA  ( IMOD, NDSE, NDST, OUTFLAGS )
 !
 #ifdef W3_CESMCOUPLED
       IF ( OUTFLAGS( 6, 14) ) THEN
-          ALLOCATE ( WADATS(IMOD)%XLANGMT(NXXX), STAT=ISTAT )
+          ALLOCATE ( WADATS(IMOD)%XUSSHX(NXXX), STAT=ISTAT )
+          CHECK_ALLOC_STATUS ( ISTAT )
+          ALLOCATE ( WADATS(IMOD)%XUSSY(NXXX), STAT=ISTAT )
           CHECK_ALLOC_STATUS ( ISTAT )
         ELSE
-          ALLOCATE ( WADATS(IMOD)%XLANGMT(1), STAT=ISTAT )
+          ALLOCATE ( WADATS(IMOD)%XUSSHX(1), STAT=ISTAT )
+          CHECK_ALLOC_STATUS ( ISTAT )
+          ALLOCATE ( WADATS(IMOD)%XUSSY(1), STAT=ISTAT )
```

Comment:
Again, I believe you mean to allocate XUSSHY here, and not ``XUSSY`.

Reply:
Fixed.